Gravity/Gravity.Test.Unit/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="NUnit" version="3.10.1" targetFramework="net451" />
```

Comment:
3.6 vs 3.10 version question?  Does it make a difference?

Reply:
We should consolidate to the newest version.

Reply:
ok, let's get the other projects updated too.

Reply:
This has been taken care of in the last commit.